### DISCUSSION TITLE
.sub to one .jpg or .png file

### DISCUSSION THREAD

**Discussion by CarltonTheDoorman** [SubtitleEdit/subtitleedit#6149] [Ideas]
Hi,
Hope a soon day we can save all .sub “text” (image) in one .JPG or .PNG file.
Because the OCR of Subtitle Edit is terrible, and it is just impossible to convert .SUB to .SRT of just if we have one month to lose to make all corrections manually.

Have one .jpg contain all “text” will be “easy” after that to convert to real text because good OCR exist.

Thanks.

**Comment by darnn** [comment]
You have several image export options in the right-click menu:
![save images](https://user-images.githubusercontent.com/19672444/184600542-cbdedc9e-e5fb-45c2-ab94-57361a64e5f0.png)

If you export all the images, I believe then you can follow the rest of the steps here (or at least do something similar):
https://youtu.be/Cd36qODmYF8?t=274
